app/controllers/members/dues_controller.rb

```diff
@@ -74,4 +75,18 @@ def redirect_target
       members_user_dues_path(current_user)
     end
   end
+
+  def plans
```

Comment:
Optional, minor: I would enjoy a more descriptive name here, maybe `amount_plan_name` or something like that.